Source/Core/Core/Analytics.cpp

```diff
@@ -278,9 +278,9 @@ void DolphinAnalytics::MakePerGameBuilder()
   // We grab enough to tell what percentage of our users are playing with keyboard/mouse, some kind
   // of gamepad
   // or the official gamecube adapter.
-  builder.AddData("gcadapter-detected", GCAdapter::IsDetected());
+  builder.AddData("gcadapter-detected", GCAdapter::IsDetected(NULL));
   builder.AddData("has-controller", Pad::GetConfig()->IsControllerControlledByGamepadDevice(0) ||
-                                        GCAdapter::IsDetected());
+                                        GCAdapter::IsDetected(NULL));
```

Comment:
This should probably use `nullptr` instead of `NULL`

Reply:
Right. Will fix.